source/documentation/process_flows.html.md

```diff
@@ -473,4 +473,17 @@ own guarantees even though no MRN may have been allocated to the transit movemen
 1. The Guarantee Management System replies back internally with the ‘Response Query on Guarantees’ E_GUA_RSP (IE037) message.
 1. The ‘Response Query on Guarantees’ E_GUA_RSP (IE037) message is converted internally to a JSON guarantee balance response, which is sent to the holder of the transit procedure.
 
+## Enquiry message flows
 
+### Enquiry with movement unknown at destination
+
+This scenario shows what happens when the competent authority of enquiry at departure has no information about a transit movement within all relevant time limits.
+
+<img src="../figures/enquiry_movement_unknown.svg" alt="Enquiry message flow. Flow is described in this section." />
+
+<a href="../figures/enquiry_movement_unknown.svg" target="_blank">Open the diagram in a new tab.</a>
+
+1. The process starts when the ‘Release for Transit’ E_REL_TRA (IE029) message is sent to the holder of the transit procedure.
+1. The competent authority of enquiry at departure does not have sufficient information that would allow for the discharge of the transit procedure and so sends a ‘Request on Non-Arrived Movement’ E_REQ_MOV (IE140) message to the holder of the transit procedure.
```

Comment:
Minor issue: "and so sends" has cadence problems. I would write either "procedure and thus sends" or "procedure, so it sends".